include/zephyr/drivers/led_strip.h

```diff
@@ -71,6 +72,14 @@ typedef int (*led_api_update_channels)(const struct device *dev,
 				       uint8_t *channels,
 				       size_t num_channels);
 
+/**
+ * @typedef led_api_length
+ * @brief Callback API for getting length of an LED strip.
+ *
+ * @see led_strip_length() for argument descriptions.
+ */
+typedef int (*led_api_length)(const struct device *dev);
```

Comment:
This value is quantity. So, It should be better unsigned value.

Reply:
Reason of it being an int is that it may return a negative errno

Reply:
As in, always return the length even if the driver init failed and never return an error from this function?

Reply:
There are implementation examples for the counter API that simply returns a fixed value without returning an error.

https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/drivers/counter.h#L294-L302

Similar to these, we expect an implementation that simply returns the value of devicetree as is, so I think it would be a good idea to follow this example.

Reply:
Fixed